CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+## Generating Mocks
+
+As of writing, the mock objects are generated in this project's `mock` directory. When a given component is altered and its test suite depends upon a mock object new mocks need to be generated in the `mock` directory to replace the now outdated mocks. 
+
```

Comment:
I think you're missing a word:  "As of _this_ writing"?

However, I would omit that phrase, because it kind of goes without saying.  Also, it's kind of apologizing for the docs being out of date...which is a bit of a pessimistic point to be starting from....

Also, I recommend limiting the line length here to about 80 columns:  markdown doesn't care, but it makes reviewing my more graceful.